src/GafferArnold/ArnoldShader.cpp

```diff
@@ -43,6 +43,7 @@
 
 #include "Gaffer/CompoundNumericPlug.h"
 #include "Gaffer/NumericPlug.h"
+#include "Gaffer/PlugAlgo.h"
```

Comment:
We don't need this include any more.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5169/commits/b7b54bcc7fa8bc97c318ecbb5b8b461e8e7ac31b